docs/imaging_model/README.md

```diff
@@ -0,0 +1,64 @@
+# Imaging and Report Model
+
+## Motivation
+
+Participants often have files in addition to genomic files ranging from
+MRI images, histology images, and reports.
+Some of these file types relate directly to the biospecimen rather than the
+participant, eg: proteomics and histology.
+In order to capture and distribute these files, we need to model them
+appropriately.
+
+## Implementation
+
+Files will need to be distributed through gen3, so the models will need to
+inherit the Indexd model similar to GenomicFiles.
```

Comment:
It would make sense to have a central File entity that listed the common elements of files/index_d terms that apply to everything, otherwise you have redundant fields. e.g of standard fields 
- file name
- latest_did 
- id
- file format
- md5
- size

Will there be more file types that don't fit in either genomic file or imaging file that other extensions would need to be made in a similar manner?  It seems weird that "files" all have different id types.

Reply:
@rosibaj We do have a common File entity. In the embedded image on README.md, the orange colored IndexdFile model has all of the common fields you're referring to. I think you have to hit the View button to actually see the rendered markdown which includes the image

Reply:
@znatty22 yes, in the code, but in the actual database and API this will be manifested as different entities, right?